Build deb file

Running installation commands like described returns an error.

```
$ make install_deps

added 17 packages, and audited 18 packages in 3s

1 package is looking for funding
  run `npm fund` for details

found 0 vulnerabilities

added 1 package, and audited 19 packages in 823ms

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
Download installer
2022-08-03 19:45:22 URL:https://e-cdn-content.dzcdn.net/builds/deezer-desktop/8cF2rAuKxLcU1oMDmCYm8Uiqe19Ql0HTySLssdzLkQ9ZWHuDTp2JBtQOvdrFzWPA/win32/x86/5.30.220/DeezerDesktopSetup_5.30.220.exe [77848448/77848448] -> "source/deezer-setup-5.30.220.exe" [1]
Extract app archive from installer
/bin/sh: 1: 7z: not found
make: *** [Makefile:19: prepare] Error 127
```

Then when I try to run `make build_deb` returns an error "Command 'build-deb' not found." I'm on Ubuntu 20.04 LTS.

--- a/README.md
+++ b/README.md
@@ -29,7 +29,15 @@ Please note that eventhough it is automatically generated, the snapcraft package
 
 ## From source
 
-You will probably need to install some things in order to generate the packages from source: `nodejs` and `npm`, and `yarn` at least; and `flatpak-builder` to build the flatpak version.
+You will probably need to install some things in order to generate the packages from source:
+
+- nodejs
+- npm
+- yarn
+- 7z by installing `p7zip` and `p7zip-full`
+- make
+- wget
+- `flatpak-builder` (only to build the flatpak version)
 
 ### Flatpak
 

